src/backend/controllers/step.controller.js

```diff
@@ -0,0 +1,27 @@
+/* Task Controller */
+const fs = require('fs');
+const path = require('path');
+
+function controller() {
+  function getById(req, res) {
+    const contents = fs.readFileSync(path.resolve(__dirname, '../data/moduleData.json'));
+    /* @todo is there a way to pass params from the router? */
+    const { moduleOId } = req.params;
```

Comment:
can you do this in one line deconstruction? 

`const { moduleOId, phaseOId, taskOId.... }`